src/main/java/nextstep/qna/domain/answer/Answers.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.qna.domain.answer;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Answers {
+
+  public static final String ANSWER_CAN_NOT_BE_DELETED = "다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.";
+  private final List<Answer> answers = new ArrayList<>();
+
+  public Answers() {}
+
+  public void add(Answer answer) {
+    answers.add(answer);
+  }
+
+  public void validateDeleteIsPossible(NsUser user) throws CannotDeleteException {
+    for (Answer answer : answers) {
+      if (answer.isNotOwner(user)) {
+        throw new CannotDeleteException(ANSWER_CAN_NOT_BE_DELETED);
+      }
+    }
+  }
+
+  public Answer get(int index) {
+    return answers.get(index);
+  }
+
+  public int size() {
+    return answers.size();
+  }
+
+  public boolean deleteAll(NsUser writer) throws CannotDeleteException {
```

Comment:
```suggestion
  public List<DeleteHistory> deleteAll(NsUser writer) throws CannotDeleteException {
```
앞선 피드백 반영한다면 deleteAll의 반환타입이 변경될 수 있겠네요~